src/views/Phrases.vue

```diff
@@ -84,68 +94,69 @@ export default {
     <h1 class="page-title">Phrases</h1>
 
     <div class="phrase-input-category-box is-centered">
-      <br>
+      <br />
       <div class="phrase-input-box">
-        <br>
+        <br />
         <strong>Input Example:</strong>
-        <br><br>
+        <br />
+        <br />
```

Comment:
If possible, could we replace the ``<br/>`` tags with css styling?

Reply:
Created a padding-small and padding-large class